inscriptions/ISic000592.xml

```diff
@@ -185,11 +190,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/><name>Licinia</name> <g ref="#interpunct">·</g> <name>Mecisto</name>
+                    <lb n="2"/><name><expan><abbr>Q</abbr><ex>uinto</ex></expan></name> <g ref="#interpunct">·</g> <name>Manlio</name> <g ref="#interpunct">·</g> <name>Viro</name>
+                    <lb n="3"/><name><expan><abbr>Q</abbr><ex>uinto</ex></expan></name> <g ref="#interpunct">·</g> <name>Manlio</name> <g ref="#interpunct">·</g> <name>Fron
+                    <lb n="4" break="no"/>toni</name> <g ref="#interpunct">·</g> <name><expan><abbr>f</abbr><ex>ilio</ex></expan></name> <g ref="#interpunct">·</g> <name>Manli<hi rend="ligature">ae</hi></name>
+                    <lb n="5"/><name><expan><abbr>f</abbr><ex>iliae</ex></expan></name>
```

Comment:
hmmm. I know this is how EDCS resolves this, but I'm not convinced; i think the first f should be filiis, then the second as filiae makes more sense, and Licinia is then the mother of all fo them.